1.3.0 release (#117)
Esri/data-collection-dotnet: RELEASE.md

# Release notes

## 1.3.0

* Updates both WPF and UWP with a new design that adapts to work well for any screen size.
    * Adds a `ModernMapPanel` custom layout panel to facilitate consistent, responsive design for UWP and WPF.
    * Refactors styles and related XAML.
    * Switches to vector icons where possible.
* Moves the .NET Core WPF project to a separate folder to prevent possible build issues.
* Updates the code called at sign out to revoke credentials from the server, when possible, and delete them locally.
* Adds support for identifying multiple items on the map simultaneously and displaying them in a list.
    * An Arcade expression defined per layer is used to create subtitles that help you differentiate identified features.
* Adds unit tests for new code added to support multiple identify results.
    * Changes were made to shared code to facilitate unit testing of view models.
* Adds a new 'About' page, which shows the app version, ArcGIS Runtime version, and a link to a page detailing licenses. The appearance of these fields can be controlled via the app's configuration.

> * Updates doc and many code comments to avoid all use of `http:`.

Reviewer: ```suggestion
* Updates doc and many code comments to access secured `https:` endpoints.
```
